src/Worker/Core/Shims/TaskOrchestrationShim.cs

```diff
@@ -31,6 +33,8 @@ public TaskOrchestrationShim(
     {
         this.invocationContext = Check.NotNull(invocationContext);
         this.implementation = Check.NotNull(implementation);
+
+        this.logger = this.invocationContext.LoggerFactory.CreateLogger(Logs.WorkerCategoryName);
```

Comment:
Similar suggestion for category name:

`Microsoft.DurableTask.Worker.Invocation.TaskOrchestrator`

Reply:
I went with simply Microsoft.DurableTask.Worker.Orchestrations per the earlier suggestion for the reason mentioned previously.